### ISSUE TITLE
Nav-Menu Submenu Indicators cannot be toggled in collapsible layout => Submenus cannot be closed after opening.

### ISSUE THREAD

**Issue by rloes** [elementor/elementor#21757]
### Prerequisites

- [X] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [X] The issue still exists against the latest stable version of Elementor.

### Description

I have been struggling to implement a mobile menu using the elementor pro nav menu widget in collapsible mode for the last two days.

The problem is, the clicks on the sub-menu indicator are handles like a normal click on the link. This leads to the fact that a sub menu cannot be closed again as the second click after expanding the sub menu always activates the link. Even if the click is on the sub-menu indicator.

I did pin down the source of the bug to the fact, that elementor is using the smartmenus option of `subIndicatorsText` to set svgs inside of the `span.sub-arrow` element. The smartmenus libary tries to identify if a click was on the sub-menu indicator in this way:

```javascript
// elementor-pro/assets/lib/smartmenus/jquery.smartmenus.js
// line 489
var subArrowClicked = $(e.target).is('span.sub-arrow')
```

The problem is, elementor places  a `<svg>` element  inside of `span.sub-arrow`.   
Now when clicking on the sub-menu indicator `$(e.target)` is often registered as the `svg` or `path` element. These are children of `span.sub-arrow` but `subArrowClicked` will evaluate to false anyway.

A solution would be to change the condition of  `subArrowClicked`, so that it is also true if a child element of `span.sub-arrow` was clicked:

```javascript
// elementor-pro/assets/lib/smartmenus/jquery.smartmenus.js
// line 489
var subArrowClicked = $(e.target).is('span.sub-arrow') || $(e.target).is('span.sub-arrow *')
```

I know, it could be argued that this issue is with the smartmenus libary and I will consider proposing a change there. However as elementor uses the subIndicatorTexts with a svg which seems  unsupported by smartmenus and elementor uses an old version from 2016 anyway. I thought opening a ticket here first is appropriate.

I would really appreciate it, if you would consider fixing this issue. The current functionality makes the nav-menu unusable for sidebar menus that do not use a dropdown under a hamburger button. 

Furthermore I do not know of a way to extend smartmenus functionality as proposed without changing the files directly in elementor-pro plugin files. This is very annyoing because I have to change the files on every elementor pro update.

### Steps to reproduce

1) add the nav-menu widget to a page
2) select a menu that has atleast one level of sub-menus
3) select "collapsible" in layout
4) select a sub menu indicator
4) open the collapsed sub menu
5) try to close it again by clicking on the sub menu indicator

### Isolating the problem

- [X] This bug happens with only Elementor plugin active (and Elementor Pro).
- [X] This bug happens with a Blank WordPress theme active ([Hello theme](https://wordpress.org/themes/hello-elementor/)).
- [X] I can reproduce this bug consistently following the steps above.

### System Info

					
== Server Environment ==
	Operating System: Linux
	Software: Apache
	MySQL version: Debian 11 v10.5.18
	PHP Version: 8.0.25
	PHP Memory Limit: 256M
	PHP Max Input Vars: 1000
	PHP Max Post Size: 150M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 6.1.1
	Site URL: localhost
	Home URL: localhost
	WP Multisite: No
	Max Upload Size: 150 MB
	Memory limit: 256M
	Max Memory limit: 256M
	Permalink Structure: /%postname%/
	Language: de-DE
	Timezone: Europe/Berlin
	Admin Email: intern@westsite-webdesign.de
	Debug Mode: Active

== Theme ==
	Name: Hello Elementor Child
	Version: 1.0.1.1676691042
	Author: Elementor Team
	Child Theme: Yes
	Parent Theme Name: Hello Elementor
	Parent Theme Version: 2.6.1
	Parent Theme Author: Elementor Team

== User ==
	Role: administrator
	WP Profile lang: de_DE_formal
	User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:109.0) Gecko/20100101 Firefox/111.0

== Active Plugins ==
	Advanced Custom Fields
		Version: 6.0.7
		Author: WP Engine

	Ajax add to cart for WooCommerce
		Version: 2.0.2
		Author: QuadLayers

	Better Elementor Excerpt
		Version: 1.0.0
		Author: Patryk Kożuch

	Child Theme Configurator
		Version: 2.6.0
		Author: Lilaea Media

	Code Snippets
		Version: 3.3.0
		Author: Code Snippets Pro

	Copy & Delete Posts
		Version: 1.3.9
		Author: Copy Delete Posts

	Custom Post Type UI
		Version: 1.13.4
		Author: WebDevStudios

	Elementor
		Version: 3.11.5
		Author: Elementor.com

	Elementor Pro
		Version: 3.11.7
		Author: Elementor.com

	Germanized for WooCommerce
		Version: 3.12.1
		Author: vendidero

	Loco Translate
		Version: 2.6.4
		Author: Tim Whitlock

	Newsletter
		Version: 7.6.8
		Author: Stefano Lissa & The Newsletter Team

	PDF Invoices & Packing Slips for WooCommerce
		Version: 3.4.0
		Author: WP Overnight

	Shortcodes Finder
		Version: 1.5.2
		Author: Scribit

	SVG Support
		Version: 2.5.5
		Author: Benbodhi

	The SEO Framework
		Version: 4.2.8
		Author: The SEO Framework Team

	WooCommerce
		Version: 7.5.1
		Author: Automattic

== Eigenschaften ==
	Custom Fonts: 1
	Custom Icons: 0

== Integration ==
	
	woocommerce: Aktiv

== Elementor Experiments ==
	Optimierte DOM-Ausgabe: Aktiv
	Verbessertes Laden von Assets: Aktiv
	Verbessertes Laden von CSS: Aktiv
	Inline Font Icons: Aktiv
	Verbesserungen der Zugänglichkeit: Aktiv
	Zusätzliche benutzerdefinierte Haltepunkte: Aktiv
	admin_menu_rearrangement: Standardmäßig inaktiv
	Flexbox Container: Aktiv
	Upgrade Swiper Library: Aktiv
	Default to New Theme Builder: Aktiv
	Hello Theme Header & Footer: Aktiv
	Editor V2: Standardmäßig inaktiv
	Startseiten: Aktiv
	Verschachtelte Elemente: Aktiv
	Lazy Load Background Images: Aktiv
	Als Standard speichern: Aktiv
	Page Transitions: Aktiv
	Notes: Aktiv
	Loop: Aktiv
	Form Submissions: Aktiv
	Scroll Snap: Aktiv
	Mega Menu: Standardmäßig inaktiv

== Log ==
	
Log: showing 20 of 272023-03-12 23:39:46 [info] elementor::elementor_updater Started 
2023-03-12 23:39:46 [info] Elementor/Upgrades - _on_each_version Start  
2023-03-12 23:39:46 [info] Elementor/Upgrades - _on_each_version Finished 
2023-03-12 23:39:46 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.11.3',
  'to' => '3.11.4',
)]
2023-03-12 23:39:46 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.11.3',
  'to' => '3.11.4',
)]
2023-03-15 13:23:17 [info] elementor::elementor_updater Started 
2023-03-15 13:23:17 [info] Elementor/Upgrades - _on_each_version Start  
2023-03-15 13:23:17 [info] Elementor/Upgrades - _on_each_version Finished 
2023-03-15 13:23:17 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.11.4',
  'to' => '3.11.5',
)]
2023-03-15 13:23:17 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.11.4',
  'to' => '3.11.5',
)]
2023-03-16 00:52:28 [info] elementor-pro::elementor_pro_updater Started 
2023-03-16 00:52:28 [info] Elementor Pro/Upgrades - _on_each_version Start  
2023-03-16 00:52:28 [info] Elementor Pro/Upgrades - _on_each_version Finished 
2023-03-16 00:52:28 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.11.5',
  'to' => '3.11.6',
)]
2023-03-16 00:52:28 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.11.5',
  'to' => '3.11.6',
)]
2023-03-24 01:39:49 [info] elementor-pro::elementor_pro_updater Started 
2023-03-24 01:39:49 [info] Elementor Pro/Upgrades - _on_each_version Start  
2023-03-24 01:39:49 [info] Elementor Pro/Upgrades - _on_each_version Finished 
2023-03-24 01:39:49 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.11.6',
  'to' => '3.11.7',
)]
2023-03-24 01:39:49 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.11.6',
  'to' => '3.11.7',
)]

JS: showing 9 of 9JS: 2023-03-08 12:58:37 [error X 14][localhost/wp-content/plugins/elementor/assets/js/frontend-modules.min.js?ver=3.11.3:2:4823] e is undefined 
JS: 2023-03-14 11:16:09 [error X 4][localhost/wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.11.4:3:676178] C.getControlModel(...) is undefined 
JS: 2023-03-22 01:07:40 [error X 5][localhost/wp-admin/load-scripts.php?c=0&loadchunk_0=underscore,jquery-core,jquery-migrate,backbone,jquery-ui-core,jquery-ui-mouse,wp-api-request,regenerator-runtime,wp-polyfill,wp-&loadchunk_1=hooks&ver=6.1.1:4:51824] Cannot read properties of undefined (reading 'replace') 
JS: 2023-03-22 15:50:38 [error X 2][localhost/wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.11.5:3:791626] this.store.find(...) is undefined 
JS: 2023-03-23 20:48:03 [error X 4][localhost/wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.11.5:3:931389] Cannot read properties of undefined (reading 'settings') 
JS: 2023-03-23 20:51:22 [error X 3][localhost/wp-content/plugins/elementor/assets/js/common-modules.min.js?ver=3.11.5:2:53736] Model.isValidChild() should be implemented, please provide 'isValidChild' functionality. 
JS: 2023-03-23 23:14:35 [error X 4][localhost/wp-admin/load-scripts.php?c=0&loadchunk_0=underscore,jquery-core,jquery-migrate,backbone,jquery-ui-core,jquery-ui-mouse,wp-api-request,regenerator-runtime,wp-polyfill,wp-&loadchunk_1=hooks&ver=6.1.1:4:51816] a is undefined 
JS: 2023-03-24 02:22:53 [error X 1][localhost/wp-content/plugins/elementor-pro/assets/js/page-transitions-editor.69f365c96dc0120de70b.bundle.min.js:2:4182] elementor.$previewContents is undefined 
JS: 2023-03-24 02:23:54 [error X 1][localhost/wp-includes/js/wp-auth-check.min.js?ver=6.1.1:2:652] h is undefined 

PHP: showing 6 of 6PHP: 2023-03-13 02:19:57 [warning X 9][/home/www/basisdruck/wp-content/plugins/elementor/core/common/modules/ajax/module.php::175] Undefined array key "data" [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-03-13 02:19:58 [warning X 4][/home/www/basisdruck/wp-content/plugins/elementor-pro/core/app/modules/site-editor/data/endpoints/templates.php::150] Undefined array key "condition_type" [array (
  'trace' => '
#0: /home/www/basisdruck/wp-content/plugins/elementor-pro/core/app/modules/site-editor/data/endpoints/templates.php(150): Elementor\Core\Logger\Manager -> rest_error_handler()
#1: ElementorPro\Core\App\Modules\SiteEditor\Data\Endpoints\Templates -> normalize_template_json_item()
#2: /home/www/basisdruck/wp-content/plugins/elementor-pro/core/app/modules/site-editor/data/endpoints/templates.php(120): class type array_map()
#3: /home/www/basisdruck/wp-content/plugins/elementor-pro/core/app/modules/site-editor/data/endpoints/templates.php(59): ElementorPro\Core\App\Modules\SiteEditor\Data\Endpoints\Templates -> normalize_templates_json()
#4: /home/www/basisdruck/wp-content/plugins/elementor/data/base/endpoint.php(158): ElementorPro\Core\App\Modules\SiteEditor\Data\Endpoints\Templates -> get_items()
',
)]
PHP: 2023-03-14 00:17:03 [warning X 150][/home/www/basisdruck/wp-content/plugins/elementor-pro/modules/woocommerce/widgets/cart.php::2383] Undefined array key "section_coupon_display" [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-03-19 19:54:22 [error X 1][/home/www/basisdruck/wp-content/plugins/elementor-pro/modules/theme-builder/classes/conditions-manager.php::295] Uncaught Error: Call to a member function update_meta() on bool in /home/www/basisdruck/wp-content/plugins/elementor-pro/modules/theme-builder/classes/conditions-manager.php:295
Stack trace:
#0 /home/www/basisdruck/wp-content/plugins/elementor-pro/modules/theme-builder/classes/conditions-manager.php(168): ElementorPro\Modules\ThemeBuilder\Classes\Conditions_Manager->save_conditions(NULL, Array)
#1 [internal function]: ElementorPro\Modules\ThemeBuilder\Classes\Conditions_Manager->ajax_save_theme_template_conditions(Array, Object(Elementor\Core\Common\Modules\Ajax\Module))
#2 /home/www/basisdruck/wp-content/plugins/elementor/core/common/modules/ajax/module.php(175): call_user_func(Array, Array, Object(Elementor\Core\Common\Modules\Ajax\Module))
#3 /home/www/basisdruck/wp-includes/class-wp-hook.php(308): Elementor\Core\Common\Modules\Ajax\Module->handle_ajax_request('')
#4 /home/www/basisdruck/wp-includes/class-wp-hook.php(332): WP_Hook->apply_filters('', Array)
#5 /home/www/basisdruck/wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#6 /home/www/basisdruck/wp-admin/admin-ajax.php(188): do_action('wp_ajax_element...')
#7 {main}
  thrown [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-03-20 00:10:31 [warning X 25][/home/www/basisdruck/wp-content/plugins/elementor-pro/modules/woocommerce/widgets/checkout.php::4110] Undefined array key "coupon_section_display" [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-03-22 14:19:30 [warning X 5][/home/www/basisdruck/wp-content/plugins/elementor-pro/modules/theme-builder/classes/conditions-manager.php::168] Undefined array key "editor_post_id" [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]

== Elementor - Compatibility Tag ==
	
	Better Elementor Excerpt: Kompatibilität nicht angegeben
	Elementor Pro: Kompatibilität nicht angegeben

== Elementor Pro - Compatibility Tag ==

**Comment by rloes** [comment]
This [plugin/snippet](https://github.com/rloes/elementor-fixed-nav-menu) provides a fix for this issue, until it is fixed.
